### ISSUE TITLE
Classes no longer work

### ISSUE THREAD

**Issue by bbashy** [spatie/laravel-menu#132]
Composer upgrade causing classes to no longer work `spatie/menu 3.0.1 => 3.1.0`.

This is not just 1 app, tested 3, all same issue. Doesn't do any of the following methods;
* ->setActiveClass()
* ->addItemClass()
* ->addItemParentClass()

```php
Menu::macro('main', function() {
    return Menu::new()
        ->withoutWrapperTag()
        ->setActiveClassOnLink()
        ->setActiveClassOnParent(false)
        ->setActiveClass('on')
        ->add(Link::to(route('dashboard'), 'Dashboard'))
        ->setActiveFromRequest()
        ;
});
```

```html
<ul>
    {!! Menu::main() !!}
</ul>
```
### Output
v3.0.1
```html
<ul>
    <li><a href="https://example.com" class="on exact-active">Dashboard</a></li>
</ul>
```
v3.1.0
```html
<ul>
    <li><a href="https://example.com">Dashboard</a></li>
</ul>
```

**Comment by el3wdy** [comment]
i have same issue.

**Comment by el3wdy** [comment]
The addition of the "addId" function to Spatie/Menu was causing a major problem. However, if the function is able to work properly, it would be an awesome feature. To make the function work.

```
    public function toArray(): array
    {
        if (empty($this->classes) || empty($this->id)) {
            return $this->attributes;
        }

        return array_merge($this->attributes, [
            'class' => implode(' ', $this->classes),
            'id' => implode(' ', $this->id),
        ]);
    }
```

```
    public function toArray(): array
    {
        if (empty($this->classes)) {
            return $this->attributes;
        }

        return array_merge($this->attributes, [
            'class' => implode(' ', $this->classes),
        ]);
    }
```

**Comment by sebastiandedeyne** [comment]
Sorry, this was my fault. I accidentally released while CI was red.

I fixed it and released `3.5.1` :)
